src/lib_json/json_value.cpp

```diff
@@ -23,18 +23,9 @@
 
 namespace Json {
 
-// This is a walkaround to avoid the static initialization of Value::null.
-// kNull must be word-aligned to avoid crashing on ARM.  We use an alignment of
-// 8 (instead of 4) as a bit of future-proofing.
-#if defined(__ARMEL__)
-#define ALIGNAS(byte_alignment) __attribute__((aligned(byte_alignment)))
-#else
-#define ALIGNAS(byte_alignment)
-#endif
-static const unsigned char ALIGNAS(8) kNull[sizeof(Value)] = { 0 };
-const unsigned char& kNullRef = kNull[0];
-const Value& Value::null = reinterpret_cast<const Value&>(kNullRef);
-const Value& Value::nullRef = null;
+static const Value kNull((Value::StaticInitTag()));
```

Comment:
The word 'static' here is redundant. namespace-scope const always have internal linkage by default.
If kNull as a (preferably private) member of Value, it would have access to struct StaticInitTag, even if that struct was also made private. I think that's a good idea to do. We might not want to publish the hackery to client code. We don't need client code relying on the ability to create a null-valued Value from a block of zeros.